### PR TITLE
Release/sim 1610 release 3.8.0 and jmp 2.5.0

### PR DESCRIPTION
**Pull request by luigiwerzowa**
This is the PR that is going to be published once JEP 3.8.0 and JMP 2.5.0 are released.

### CODE REVIEW

File: docs/rules/remove-null-check-before-instanceof.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Java 1.1
+:::
+
+::: tip Tags
+<TagLinks />
+:::
+
+## Code Changes
+
+### Null-check in conjunction with instanceof
```
Comment (arditymeri) [inline docs/rules/remove-null-check-before-instanceof.md]: In the description, null check is written without dash (-). Do we want to remove it from here too?

File: docs/rules/use-collections-singleton-list.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
+title: Use Collections Singleton List
+description:
+    Replaces Arrays.asList with 0 or 1 parameters respectively with Collections.emptyList(...) or Collections.singletonList()
```
Comment (arditymeri) [inline docs/rules/use-collections-singleton-list.md]: Oops.. I think I meant Collections.emptyList() and Collections.singletonList(..). The dots are wrong.

File: docs/rules/use-collections-singleton-list.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+| Property                        | Value |
+|:------------------------------- |:----- |
+| First seen in jSparrow version  | [3.8.0](/eclipse/release-notes.html#_3-8-0) |
+| Minimum Java version            | 1.3   |
+| Remediation cost                | 2 min |
+
+## Description
+
+Replaces [`Arrays.asList`](https://docs.oracle.com/javase/8/docs/api/java/util/Arrays.html#asList-T...-) with 0 or 1 parameters respectively with [`Collections.emptyList()`](https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#emptyList--) or [`Collections.singletonList(..)`](https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#singletonList-T-).
```
Comment (arditymeri) [inline docs/rules/use-collections-singleton-list.md]: here the dots are ok

File: docs/rules/remove-null-check-before-instanceof.md (hunk 1 above)
Comment (luigiwerzowa) [reply docs/rules/remove-null-check-before-instanceof.md]: Changed all occurrences of "null check" to "null-check"

File: docs/rules/use-collections-singleton-list.md (hunk 2 above)
Comment (luigiwerzowa) [reply docs/rules/use-collections-singleton-list.md]: Fixed it.

**Pull request merged**